vars/rollingUpgradePipeline.groovy

```diff
@@ -86,14 +92,27 @@ def call(Map pipelineParams) {
                             tasks["${base_version}"] = {
                                 node(builder.label) {
                                     withEnv(["AWS_ACCESS_KEY_ID=${env.AWS_ACCESS_KEY_ID}",
-                                             "AWS_SECRET_ACCESS_KEY=${env.AWS_SECRET_ACCESS_KEY}",]) {
+                                             "AWS_SECRET_ACCESS_KEY=${env.AWS_SECRET_ACCESS_KEY}",
+                                             "SCT_TEST_ID=${UUID.randomUUID().toString()}",]) {
 
-                                        stage("Upgrade from ${base_version}") {
+                                        stage("Create SCT Runner for ${base_version}") {
                                             catchError(stageResult: 'FAILURE') {
                                                 wrap([$class: 'BuildUser']) {
                                                     dir('scylla-cluster-tests') {
                                                         checkout scm
+                                                        timeout(time: 5, unit: 'MINUTES') {
```

Comment:
@bentsi , @amoskong  Is converting `rollingUpgradePipeline.groovy` to use runners part of the PR ?

Reply:
Yes.

Reply:
The upgrade tests and artifacts-tests are the major part to use GCE backend.
We want to switch them to use (gce) sct runner.